acsoo/addons.py

```diff
@@ -80,31 +80,28 @@ def addons_list_depends(ctx, exclude):
 
 @click.command()
 @click.option('--database')
-@click.option('--odoo-bin', 'odoo_bin', default=None)
+@click.option('--odoo-bin', 'odoo_bin', default='odoo')
 @click.option('--odoo-config', 'odoo_config',
               type=click.Path(dir_okay=False, exists=True))
 @click.option('--git-commit', 'git_commit', is_flag=True, default=False)
-@click.option('--languages', default='')
 @click.option('--git-push', 'git_push', is_flag=True, default=False)
-@click.option('--languages', default='')
+@click.option(
+    '--languages', default='',
```

Comment:
Let's rename this option `--create-languages' help="Comma separated list of languages for which the .po files will be created if not present"

Reply:
But the option in acsoo.cfg can remain `languages`.

Reply:
Is the languages option in acsoo.cfg still needed? If we loop on existing languages and if we have a --create-languages option I think it's enough. And we can create new translations from weblate.